test/unit/legacy_unit/cons/editor.c

```diff
@@ -1,6 +0,0 @@
-#include <r_cons.h>
```

Comment:
this could be `rvi` in extras :P

Reply:
`rvi`?

Reply:
radare visual (editor) like vi, but with r2